### PR TITLE
Add canAny passthrough method.

### PR DESCRIPTION
**Pull request by hailwood**
This PR is in relation to questions asked on larachat by @emilas44

### CODE REVIEW

**Comment by emilas44** [comment]
@JosephSilber  can you please merge this?

File: readme.md [hunk 1]
```diff
@@ -547,11 +547,16 @@ For convenience, the bouncer class provides these passthrough methods:
...
+Bouncer::canAny($arrayOfAbilities);
+Bouncer::canAny($arrayOfAbilities, $model);
 Bouncer::cannot($ability);
+Bouncer::cannot($ability, $model);
 Bouncer::authorize($ability);
+Bouncer::authorize($ability, $model);
 ```
 
-These call directly into the `Gate` class.
+These call directly into the `Gate` class, so bouncer abilities can also be checked directly on the gate e.g. `Gate::can($ability)`.
```
Comment (JosephSilber) [inline readme.md]: I don't believe this belongs here.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 5.5.9
-  - 5.5
```
Comment (JosephSilber) [inline .travis.yml]: Can you please rebase? We don't even have this file anymore.

**Comment by hailwood** [comment]
@JosephSilber my fork was so out of date I just nuked it and redid the PR

https://github.com/JosephSilber/bouncer/pull/510

**Pull request closed without merging**